Makefile

```diff
@@ -18,6 +18,12 @@ CYN="$$(tput setaf 6)"
 # reset terminal output
 NRM="$$(tput sgr0)"
 
+# Go compiler
+GO := go
```

Comment:
Any particular reason for making this a variable? If not let's remove and just call go directly in the targets.